packages/earlybird/earlybird.0.1.1/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+name: "earlybird"
+version: "0.1.1"
```

Comment:
The version field is no longer used and will raise a warning. I'll remove it

Reply:
I take it back, I have double checked in the documentation, the warning is also when (if present) they don't match the name and version of the package inferred from the path